docker-amundsen-atlas.yml

```diff
@@ -10,7 +10,7 @@ services:
       environment:
         - ATLAS_PROVISION_EXAMPLES=true
   amundsensearch:
-      image: amundsen-search:latest
+      image: photoft/amundsen-search:latest
```

Comment:
What is `photoft`?

Reply:
that's my docker id.

Reply:
Just created `amundsendev` account and let's use that one.

Reply:
@jinhyukchang @feng-tao I like that `amundsendev` better, and noticed you already used that user account name on pypi. 

But if it’s not too late already I’d like to propose `meet-amundsen` or maybe just`amundsen` which strangely enough hasn’t been taken on docker hub ... 

I can’t take credit for the “brand idea” though 😄 the source is slide 16 in https://www.slideshare.net/mobile/taofung/data-council-sf-amundsen-presentation which I think makes up for a great unique slogan +it’s relatively easy to google just like “get Firefox” .... but if you google for “get Amundsen” you will see it’s already a bit crowded partly because of the Norwegian http://get.no news domain.

Thoughts?

Reply:
we will switch to use amundsendev docker account :) Thanks for @jinhyukchang for setting it up.

Reply:
I will update the pr once all the new images are published under that account.

Reply:
@feng-tao Just published images into `amundsendev`
https://hub.docker.com/u/amundsendev

Reply:
update

Reply:
@jinhyukchang pr updated